TLM/TLM/Custom/AI/CustomCarAI.cs

```diff
@@ -174,7 +175,7 @@ public class CustomCarAI : CarAI {
             IExtVehicleManager extVehicleMan = Constants.ManagerFactory.ExtVehicleManager;
             extVehicleMan.UpdateVehiclePosition(vehicleId, ref vehicleData);
 
-            if (!Options.isStockLaneChangerUsed()
+            if (!Options.IsStockLaneChangerUsed()
```

Comment:
Can we rename this - it's only used in 3 places (`CustomCarAI`, `CustomTrainAI` and `CustomTramBaseAI`)...

Definition in `Options.cs`:

```csharp
internal static bool isStockLaneChangerUsed() {
    return !advancedAI;
}
```

Everywhere it's used:

```csharp
if (!Options.IsStockLaneChangerUsed() ... )
```

Why not rename to `Options.IsAdvancedAiUsed()` then we can ditch the extraneous `!` both in definition and usage of the method:

```csharp
internal static bool isAdvancedAiUsed() {
    return advancedAI;
}

// some time later
if (Options.IsAdvancedAIUsed() ... )
```

Also, what's the reason for a method call - why not just reference `Options.advancedAI` directly?

Reply:
Done